builtin/providers/openstack/resource_openstack_lb_loadbalancer_v2.go

```diff
@@ -57,6 +57,12 @@ func resourceLoadBalancerV2() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"vip_port_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Thanks, Paul. I was going to double-check this on Slack :)

So just remove the `Optional`?